backend/layers/thirdparty/s3_provider.py

```diff
@@ -52,13 +57,14 @@ def delete_files(self, bucket_name: str, object_keys: List[str]):
         """
         Deletes the objects `object_keys` from bucket `bucket_name`
         """
-
-        def chunk_list(items: List[str], chunk_size: int = 1000):  # s3 client delete_objects accepts up to 1000 keys
-            for i in range(0, len(items), chunk_size):
-                yield items[i : i + chunk_size]
-
-        for keys_batch in chunk_list(object_keys):
-            self.client.delete_objects(Bucket=bucket_name, Delete={"Objects": [{"Key": key} for key in keys_batch]})
+        for i in range(0, len(object_keys), AWS_S3_MAX_ITEMS_PER_BATCH):
+            key_batch = object_keys[i:AWS_S3_MAX_ITEMS_PER_BATCH]
+            logger.info(f"Deleting assets from public-access bucket '{bucket_name}': {key_batch}")
+            self.client.delete_objects(
```

Comment:
We should also log the Deleted and Errors fields from this response.

Reply:
The value mentioned above are hard to find after deleting the files. Since deletion is a sensitive topic, I'd rather error on the side of caution and log more than we might need.